src/content/docs/mobile-monitoring/new-relic-monitoring-react-native/react-native-installation.mdx

```diff
@@ -0,0 +1,312 @@
+---
+title: React Native agent
+tags:
+  - Mobile monitoring
+  - New Relic Mobile
+  - Configuration
+metaDescription: Instructions for installing and configuring the New Relic React Native agent.
+---
+
+With our React Native agent, your team can monitor your hybrid app's performance and identify any errors within your code. Our agent collects crash data, network traffic, and other information for hybrid apps using native components. This allows your team to better understand the health of your app. We empower your team to make more informed development choices by providing insight into the following:
+
```

Comment:
think the best formatting for bulletpoints is to a) have no line break between them, and b) make the asterisks flush left. (I could be wrong but that's how I do it anyway)

Reply:
Same